src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/RemoveStartArgumentsActionHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Xml;
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[Export(typeof(ILinkActionHandler))]
+[ExportMetadata("CommandName", "RemoveStartArguments")]
+internal class RemoveStartArgumentsActionHandler : ILinkActionHandler
+{
+    private const string PropertyNameToDelete = "StartArguments";
+    private const string UserSuffix = ".user";
+
+    public RemoveStartArgumentsActionHandler()
+    {
+    }
+
+    public Task HandleAsync(UnconfiguredProject project, IReadOnlyDictionary<string, string> editorMetadata)
+    {
+        string xmlFile = project.FullPath + UserSuffix;
```

Comment:
fixed in last commit

Reply:
> In fact we don't even have to wait for that async work to complete. We could just _ = JTF.RunAsync(RemoveElementAsync); and not await that task.

Looking at your change, it's possible I gave you bad advice here. It might be worth making the user wait for this operation to complete, to prevent race conditions with subsequent user actions.

I still think we should switch to the thread pool. See my more recent comment.